### PR TITLE
Changes made to homework for ChooseYourOwnAdventure.

### PR DESCRIPTION
**Pull request by mvaden**
Completeness: 2
Comfort: 2.5

### CODE REVIEW

**Comment by tylercrosse** [comment]
I love seeing the psuedocode! You are definitely on the right track with assignment. I really encourage you to continue to work on this a little more and make another commit. I'll leave the pull request open for the time being. Please let me know if you're having trouble with a particular part of it!

**Comment by mvaden** [comment]
Update: Choose Your Own Adventure In ruby.
Thanks for your help, Tyler!
Only problem was a syntax error on line 80, col 1. No other noted errors.
Comfort Level: 4/5
Completed-ness: 4.8/5

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+   elsif dance == "n"
+      puts "Honorable. But this also means that your future Dad will never meet your Mom, and therefore you cannot exist."
+
+   elsif dance == "s"
+      puts "Interesting. You set up an elaborate plan for your future Dad to surprise your Mom by beating you up. Despite going horribly awry, the plan ultimately works. You may go back to your own time."
+
+   end # y n s
+
+end # 1955 - 1984
```
Comment (tylercrosse) [inline app.rb:77]: Line77 is the source of your error. We can't start an `elsif` without an if statement before it. Here are some valid formats for conditionals, notice they all only say `end` once and comes at the end (named that for a reason)

``` rb
#example 1
if (condition)
  # do stuff
end

#example 2
if (condition)
  # do stuff
else
  # do other stuff
end

#example 3
if (condition)
  # do stuff
elsif (another condtion)
  # do other stuff
end

#example 4
if (condition)
  # do stuff
elsif (another condtion)
  # do other stuff
else
  # some other other stuff
end
```

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+      moment == gets.chomp
+      puts moment
+
+      if moment == "t"
+         puts "Interesting choice. Unfortunately the Doc can't grab Clara and get back to the car in time. He ends up staying in 1855 with her."
+
+      elsif moment == "l"
+         puts "Smart choice. Unfortunately the Doc was deeply in love with Clara, and when he gets back to 1985 he becomes very depressed."
+
+   end # t l
```
Comment (tylercrosse) [inline app.rb:106]: This end should be indented over to match the if statement that started this block

File: app.rb [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+end # h m t
```
Comment (tylercrosse) [inline app.rb:108]: This one should also be indented over to match the if statement for `h`. You also then need to include an end statement to finish the block of conditionals for years.

File: app.rb [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+   end # y n s
+
+end # 1955 - 1984
+
+
+elsif year.to_i < 1955
+   puts "I see you're a fan of Back to the Future 3. You've run out of nitroglycerin to get back to your own time. How do you power the Time Machine? **(H/M/T)."
+
+
+   power == gets.chomp
```
Comment (tylercrosse) [inline app.rb]: This won't work. Care to guess why?

Comment (tylercrosse) [inline app.rb:84]: This won't work. Care to guess why?

File: app.rb [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+   if power == "h"
+      puts "Good idea, but no. The time machine needs to get to 88mph. 12 horsepower ain't gonna cut it."
+
+   elsif power == "m"
+      puts "You'd be better off drinking the moonshine. Do not pass Go, do not collect $200. Stuck in 1855."
+
+   elsif power == "t"
+      puts "Good call! This plan seems to be working. But wait! Clara wants to go Back to the Future with you at the last moment. What do you do? **(T/L)**"
+
+      moment == gets.chomp
```
Comment (tylercrosse) [inline app.rb:97]: Same issue as line 84

**Comment by tylercrosse** [comment]
Nice work! :+1:  You need to really pay attention to what the error messages are telling you and resolve them before you continue to write other code. Otherwise you have no way of knowing if any additional code you write works or not. I would love for you to go back and fix all of these issues over the weekend and resubmit it on Monday. You should be able to get this to work all the way through.

**Pull request closed without merging**